TypeCobol.Test/Parser/Programs/Cobol85/EmptyArray.Mix.txt

```diff
@@ -0,0 +1,292 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Pgm2358.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+
+      *********************************************
+      *A. Usage is declared on array
+      *********************************************
+
+       01 Group-A.
+Line 11[15,20] <27, Error, Syntax> - Syntax error : A group item cannot be empty.
+           05 simple.
+           05 simplePicture pic X.
+      *Ok
+           05 Array-A01-OK OCCURS 50 pic X.
+
+      *A"PICTURE"clause was not found for elementary item"Array-A02-KO".
+      *   "PICTURE X(1)"was assumed.
+Line 18[15,26] <27, Error, Syntax> - Syntax error : A "PICTURE"clause was not found for elementary item "Array-A02-KO"."PICTURE X(1)"was assumed.
```

Comment:
I'm not a big fan of IBM compiler error messages, what about:
'OCCURS cannot be used on empty data item. 'Array-A02-KO' must have at least one child or declare a PICTURE.' ?

To match more closely the IBM compiler error, we can use 'A "PICTURE" clause was not found for elementary item "Array-A02-KO".', omitting the '"PICTURE X(1)"was assumed' part which is not correct in our parser.

Reply:
Done